scripts/galera_new_cluster.sh

```diff
@@ -21,11 +21,11 @@ EOF
     exit 0
 fi
 
-systemctl set-environment _WSREP_NEW_CLUSTER='--wsrep-new-cluster' && \
+echo _WSREP_NEW_CLUSTER='--wsrep-new-cluster' > @mysqlunixdir@/"wsrep-new-cluster-${1:-mariadb}" && \
```

Comment:
was the drop in execute permissions on this file intentional? Handled in CMakeList.txt, but asking anyway.

Reply:
What execute permissions? Can you elaborate?

Reply:
![image](https://github.com/MariaDB/server/assets/462287/f919f88c-7cc2-4bdf-8117-8b68dcbe8114)

Reply:
Ah, got it. Good question... Probably I've wanted to rename it to `galera_new_cluster.sh.in` as it is a template now...